### PR TITLE
[MIG] 12.0 payment terminal

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

**Review by legalsylvain — CHANGES_REQUESTED** [review]
sorry @PierrickBrun but your PR is quite hard to review because you blackified all the module.

1) for the time being, black is not an OCA convention.
2) if it become an OCA convention, please blackify in a separate commit to make easy the review.

thanks.

**Comment by PierrickBrun** [comment]
@legalsylvain I've divided it into two commits

File: pos_payment_terminal/__manifest__.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'POS Payment Terminal',
-    'version': '10.0.0.1.1',
+    'version': '12.0.0.1.0',
     'category': 'Point Of Sale',
     'summary': 'Manage Payment Terminal device from POS front end',
-    'author': "Aurélien DUMAINE,Akretion,Odoo Community Association (OCA)",
+    'author': "Akretion, Odoo Community Association (OCA)",
```
Comment (legalsylvain) [inline pos_payment_terminal/__manifest__.py]: why did you removed the author ?

File: pos_payment_terminal/static/src/xml/pos_payment_terminal.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <t t-extend="PaymentScreen-Paymentlines" >
         <t t-jquery=".col-name" t-operation="append">
             <t t-if="line.cashregister.journal.payment_mode and widget.pos.config.iface_payment_terminal">
-                   <button class="payment-terminal-transaction-start"  t-att-data-cid='line.cid'>Start transaction</button>
+                   <button class="payment-terminal-transaction-start"  t-att-data-cid='line.cid'>Send to terminal</button>
```
Comment (rousseldenis) [inline pos_payment_terminal/static/src/xml/pos_payment_terminal.xml:6]: Why this change? IMHO user 'functionnaly' understands more 'Start transaction' than send one. But it's my feeling

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Code review & test

minor remarks. on journal, it seems that it is missing a group. The field is not good displayed : 

image

more important problem, if the iot box / pywebdriver is not responding, I can not confirm the order. Did you faced the same problem ?

image

regards.

**Comment by PierrickBrun** [comment]
@legalsylvain I've been able to reproduce, this needs some refactoring I think

File: pos_payment_terminal/__manifest__.py (hunk 1 above)
Comment (PierrickBrun) [reply pos_payment_terminal/__manifest__.py]: I've left only the organisations in this field, isn't it how it is supposed to work ? Aurélien and Alexis are on the top of the file

File: pos_payment_terminal/static/src/xml/pos_payment_terminal.xml (hunk 2 above)
Comment (PierrickBrun) [reply pos_payment_terminal/static/src/xml/pos_payment_terminal.xml:6]: I thought it was easier to understand but this is only my point of view. Not changing it should be better for the users in case of migrations

**Comment by legalsylvain** [comment]
@PierrickBrun : any news on that PR ? 

regards.

**Comment by PierrickBrun** [comment]
@legalsylvain None at all and I don't think I will continue working on this since I realized the hardware I wanted to use it with is not compatible with https://github.com/akretion/pypostelium

**Comment by legalsylvain** [comment]
supersed by https://github.com/OCA/pos/pull/440
